core/src/com/mygdx/insanerpg/Unit.java

```diff
@@ -0,0 +1,15 @@
+package com.mygdx.insanerpg;
+
+class Unit {
+    public int health;
+    private String _name;
+    private int _damage;
+    private int _level;
+    private long _uniqueID;
```

Comment:
ID should be Id, and no need for unique because Ids are already known to be unique.